submissions/madmaxWMFU/Friends-App/js/app.js

```diff
@@ -0,0 +1,100 @@
+const allFriends = {
+    currentList: [],
+    changeList: [],
+    countFriends: 52,
+    minAge: 0,
+    maxAge: 150
+};
+
+const friendZone = document.querySelector(".friends-zone");
+const filters = document.querySelector(".filters-input");
+const sorts = document.querySelector(".search-sort-options");
+const resetFilters = document.querySelector(".reset-button");
+
+const getDataApi = async () => {
+    const response = await fetch(`https://randomuser.me/api/?results=${allFriends.countFriends}`);
+    const data = await response.json();
```

Comment:
Please, handle network errors - https://www.tjvantoll.com/2015/09/13/fetch-and-errors/ It is a good thing for any application